src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -513,8 +530,29 @@ private function getResourcePaginatedCollectionType(string $resourceClass, Graph
         return $this->graphqlTypes[$resourceClass]['connection'] = new ObjectType($configuration);
     }
 
-    private function isCollection(Type $type): bool
+    private function isCollectionOfObjects(Type $type): bool
+    {
+        return $type->isCollection()
+            && Type::BUILTIN_TYPE_OBJECT === $type->getBuiltinType();
+    }
+
+    private function isCollectionOfScalars(Type $type): bool
     {
-        return $type->isCollection() && Type::BUILTIN_TYPE_OBJECT === $type->getBuiltinType();
+        return $type->isCollection()
```

Comment:
Could you please replace this by:
```
return $type->isCollection()
        return $type->isCollection()
            && null !== $type->getCollectionValueType()
            && null !== $type->getCollectionValueType()
            && null === $type->getCollectionValueType()->getClassName();
```
Unless it doesn't work of course.

Reply:
updated.
Note : the unitary test succeeded in local but failed during the "ci/circleci: phpunit-coverage" step due to a slowest test.